components/home/__tests__/homepage-header.test.tsx

```diff
@@ -10,17 +10,10 @@ describe("HomepageHeader", () => {
-/>
-`);
+    expect(headerImage).toHaveAttribute(
+      "src",
+      "https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_3840,q_90,cs_tinysrgb,f_auto/v1627049958/hero_banner_desktop_zat71c",
+    );
```

Comment:
For this particular case, I think it's fine for us to keep using `.toMatchInlineSnapshot()` because the element is quite granular, and the snapshot result is still completely readable and makes sense for us.

Reply:
i see, yeah so previously I just think that the `img` tag rendered a base64 for `srcset` attribute its kinda hard to readable the assertion 😅––but sure I will try to keep that, thanks for the input mas